src/analyze-trace-file.ts

```diff
@@ -52,6 +53,7 @@ interface EventSpan {
     start: number;
     end: number;
     children: EventSpan[];
+    droppedChildCount: number;
```

Comment:
Events under `minDuration` are dropped entirely, making it hard to approximate how many children a `findSourceFiles` event actually has.  This fix was incomplete because the compiler uses sampling to drop short events from the trace before we even get it.